### ISSUE TITLE
export gene X cell 

### ISSUE THREAD

**Issue by abehd1** [dviraran/xCell#50]
Hello there! 

Thank you for this awesome R package. I have a question regarding exporting files. I have run xCell on bulk RNA seq data. The usual result gives me a matrix of enriched cell types across samples. However, I want a matrix of enriched cell type across the genes that were used in the process. Is there a way to export the enrichment score across the used genes and not samples? 

Thanks again.

**Comment by dviraran** [comment]
Thanks!

I am not sure I understand - what is enriched cell types across genes?
If you are looking for the reference signatures you can find them in xCell.data$signatures.

Best,
Dvir

**Comment by abehd1** [comment]
Thank you for the prompt response!  Can I just use the unique gene  identifiers in the reference signature as  'markers' for the cell types?

**Comment by dviraran** [comment]
Yes, those are not perfect markers, but as a group they are supposed to be markers.
